### ISSUE TITLE
New branding request: Forms is missing an H1

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#1961]
The request form is set with an H2 instead of an H1

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/d5ea8d39-a05b-4d3a-b2a4-ebb43863d6f9)

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/95c5119a-1dd2-47d7-bfad-17a38b236e29)

The confirmation page is also missing an H1, no screenshot is available.
